### PR TITLE
Add article footer to support press images

### PR DESCRIPTION
**Pull request by gustaveen**
## What?
Add article footer to support adding other blocks and downloadable press images.

## Why?
Being able to add other blocks after the article. 

BulletPointBlock is used to create the image and download link in this example

https://user-images.githubusercontent.com/6661511/211750883-2eb68542-3104-4451-a247-26c4251385df.mov

**Ticket(s): [GRW-2048]**

[GRW-2048]: https://hedvig.atlassian.net/browse/GRW-2048?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* master
  * **PR #700** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-next/700" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-next/700?utm_source=stack-comment).

File: src/pages/ArticlePage.tsx [hunk 1]
```diff
@@ -11,6 +16,17 @@ import {
 } from '../storyblok/StoryContainer'
 import { getMeta } from '../utils/meta'
 
+const Wrapper = styled.div`
+  max-width: 41rem;
+  margin: 1rem auto 8rem;
+  padding-inline: 1rem;
+`
+const ArticleFooter = styled.div`
+  ${ContentWrapperStyled} {
```
Comment (gustaveen) [inline src/pages/ArticlePage.tsx:25]: Need to remove padding on nested blocks

File: src/pages/ArticlePage.tsx [hunk 2]
```diff
@@ -40,9 +56,32 @@ export const ArticlePage = ({ nonce }: { nonce?: string }) => {
...
+            {story && <ArticleBlock story={story} />}
+
+            <ArticleFooter>
+              {story?.content.article_footer.map((block, index) => {
+                const BlockComponent:
+                  | React.ComponentType<BaseBlockProps & any>
+                  | undefined = getBlockComponent(block.component)
+                if (!BlockComponent) {
+                  return null
+                }
```
Comment (robinandeer) [inline src/pages/ArticlePage.tsx:69]: nit: this is becoming hard to read -- consider breaking out into separate component

**Review by robinandeer — APPROVED** [review]
Cool cool

File: src/pages/ArticlePage.tsx (hunk 2 above)
Comment (gustaveen) [reply src/pages/ArticlePage.tsx:69]: indeed! will take another stab at integrating the `react` SDK if it's not too much to change with all blocks 🙌

**Pull request merged**